salt-mode.el

```diff
@@ -54,12 +54,23 @@
 ;; C-c % %
 ;;   for {# and {% as well.
 
+;; In-Emacs documentation: If you have Salt installed on the same
```

Comment:
Oh I see, that's odd that it only works for built-in packages.

Including something in the mode docstring would be good, and perhaps something in the README? I want the mode to present well when people are browsing github for tools to edit salt code with.

No worries if you'd rather not, I can add this to the README myself. Lack of docs certainly won't stop me merging this pull request.